README.rst

```diff
@@ -58,6 +58,62 @@ running a version lower than 2.0.
 
      ckan.harvest.mq.type = redis
 
+7. If you want your ckan harvest logs to be exposed to the CKAN API you need to properly 
```

Comment:
Can you move all DB logger configuration steps (7 to 9) to a separate section after "Configuration", just to make clear that this is an optional step.